src/Jobs/MigrateGTIN.php

```diff
@@ -22,6 +23,12 @@
  */
 class MigrateGTIN extends AbstractBatchedActionSchedulerJob implements OptionsAwareInterface {
 	use OptionsAwareTrait;
+	use GTINMigrationUtilities;
+
+	public const GTIN_MIGRATION_COMPLETED   = 'GTIN_MIGRATION_COMPLETED';
+	public const GTIN_MIGRATION_STARTED     = 'GTIN_MIGRATION_STARTED';
+	public const GTIN_MIGRATION_READY       = 'GTIN_MIGRATION_READY';
+	public const GTIN_MIGRATION_UNAVAILABLE = 'GTIN_MIGRATION_UNAVAILABLE';
```

Comment:
I can understand the use of the naming within the code, but for all saved options we generally use a lower case word (without prefix). The option name is already fully prefixed as `gla_gtin_migration_status`, so can we set it as values of `completed`, `started`, `ready`, `unavailable`.